src/extras/cacheAsBitmap.js

```diff
@@ -13,7 +13,7 @@ DisplayObject.prototype._cacheData = false;
  * @class
  * @ignore
  */
-class CacheData
+export default class CacheData
```

Comment:
Does this need to be exported? I didn't export it because it seemed to only be used in this file.